lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -40,6 +42,16 @@ function divideByGCD(fraction) {
     return Node.Status.noChange(fraction);
   }
 
+  const intermediateNumerator = Node.Creator.operator(
+    '/', [Node.Creator.constant(numeratorValue), Node.Creator.constant(gcd)]);
+  const intermediateDenominator = Node.Creator.operator(
+    '/', [Node.Creator.constant(denominatorValue), Node.Creator.constant(gcd)]);
+  const intermediateFraction = Node.Creator.operator(
+    '/', [intermediateNumerator, intermediateDenominator]);
+  const status = Node.Status.nodeChanged(
+    ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  substeps.push(status);
```

Comment:
ah, cause `fraction` and `intermediateFraction` aren't at all part of the nodes used in the next step? Yeah, it's probably fine then

changeGroups are kinda messy in general heh and not well tested so I wouldn't worry too much about it for now (but thinking about them a bit when you can is still good)